file-formats/aobj/aobj-v1.json

```diff
@@ -78,12 +78,19 @@
                     "description": "Name of executable class or program which reloads the data to the tables (may be empty)",
                     "type": "string",
                     "maxLength": 30
+                },
+                "storageClass": {
+                    "title": "Storage Class",
+                    "description": "Name of class which stores and retrieves archived data. The class needs to implement the interface IF_ARCH_STORAGE_MANAGER.",
```

Comment:
@schneidermic0 do you have an option on the reference to the `INTF`? Typically, the description does not refer to technical details.

Reply:
I think this information is fine for me.